utils/build-script

```diff
@@ -839,8 +891,35 @@ the number of parallel build jobs to use""",
     for v in ['MAKEFLAGS', 'SDKROOT', 'MACOSX_DEPLOYMENT_TARGET', 'IPHONEOS_DEPLOYMENT_TARGET']:
         os.environ.pop(v, None)
 
+    if args.show_sdks:
+        swift_build_support.debug.print_xcodebuild_versions([
+            'iphonesimulator',
+            'appletvsimulator',
+            'watchsimulator',
+        ])
+
     check_call(build_script_impl_args)
 
+    if args.symbols_package:
+        symbols_package_path = os.path.abspath(args.symbols_package)
```

Comment:
I feel like requiring the path to be absolute was more of a safety net.  When creating various packages, the build script is usually running in some sort of automated environment, where the current working directory is not always clear, so it was better to always require absolute paths.